lib/translator/locale-parser.js

```diff
@@ -18,14 +17,12 @@ function getPreferredLocale(acceptLanguage, languages, defaultLocale) {
     const locales = getLocales(acceptLanguage);
     let preferredLocale = defaultLocale;
 
-    _.each(locales, locale => {
-        if (languages[locale]) {
-            preferredLocale = locale;
-
-            return false;
+    for (let i = 0; i < locales.length; i++) {
+        if (languages[locales[i]]) {
+            preferredLocale = locales[i];
+            break;
         }
```

Comment:
maybe you can use Array.foreach just a suggestion

Reply:
Issue with forEach is that it doesn't have the ability to break.